src/main/java/com/es/phoneshop/Cart/Cart.java

```diff
@@ -1,9 +1,16 @@
-package com.es.phoneshop.Cart;
-
-import com.es.phoneshop.model.product.Product;
-import javax.servlet.http.HttpServletRequest;
-
-public interface Cart {
-    CartList getCart(HttpServletRequest request);
-    void addProduct(CartList cart, Product product, int quontity); 
-}
+package com.es.phoneshop.Cart;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cart {
+    private List<CartItem> cart;
+    
+    public Cart(){
+        cart = new ArrayList<>();
+    }
+    
+    public List<CartItem> getList(){
```

Comment:
Here and in other places - coding style guides:

getList(){ --> getList() {
if (error!=null){ --> if (error != null) {

Style guides: https://google.github.io/styleguide/javaguide.html

Let IDEA format the file for you: https://www.jetbrains.com/help/idea/reformat-file-dialog.html

Ctrl + Shift + Alt + L or appropriate option in settings